site/src/scss/main.scss

```diff
@@ -11,6 +11,7 @@
 
 @import "../fonts/fontello/css/fontello.css";
 
+@import "init";
```

Comment:
Can we move this up to the head of this file and replace the variables and foundation import there?